### PR TITLE
1814 - Reshape wide SLV job role columns into rows

### PR DESCRIPTION
**Pull request by MarionJHolloway**
## Description
Trello ticket [#1814](https://trello.com/c/SL6hUcbc/1814-l-pivot-6-pivot-job-role-columns-to-rows)

Implements `reshape_job_role_cols_to_rows()` in the SLV prepare job, reshaping the wide per-published-job-role `{label}_{emp,strt,stop,vacy}` columns into one row per `(establishment_id, ascwds_workplace_import_date, job_role_label)`. Updates `_01_merge.py` to read the new fixed-shape output directly (no longer needs a column selector), and reworks `validate_00_prepare.py`'s checks (grain-uniqueness, distinct-value/row-count checks) to match the new long-format output.

## Testing
- [x] Unit tests passing
- [x] Successful [Step Function run](https://eu-west-2.console.aws.amazon.com/states/home?region=eu-west-2#/v2/executions/details/arn:aws:states:eu-west-2:856699698263:execution:1814-slv-reshape-Ind-CQC-SLV:d4550652-5b99-4e08-a9fe-296155a7524e)
- [x] Outputs checked in Athena (Example output attached to trello ticket)

## Checklist (delete if not relevant)
- [x] Unit tests added/amended
- [x] Docstrings added/updated
- [x] Updated CHANGELOG
- [x] Code reviewed by AI
- [x] Moved Trello ticket to PR column

## Reviewer checklist
- [ ] PR solves the Trello ticket requirements
- [ ] Outputs appear reasonable and understandable
- [ ] The overall approach is appropriate and not over-engineered
- [ ] No obvious scalability, performance or interdependency concerns
- [ ] Tests appropriately cover the main behaviour/edge cases
- [ ] Naming and structure are broadly understandable and consistent
- [ ] Docstrings are sufficient for future maintenance
- [ ] Docstrings cover non-obvious behaviour

### CODE REVIEW

File: projects/_07_workforce_characteristics/_01_starters_leavers_vacancies/fargate/utils/prepare_utils.py [hunk 1]
```diff
@@ -124,12 +129,64 @@ def reduce_to_published_roles(lf: pl.LazyFrame) -> pl.LazyFrame:
...
     """
-    pass
+    metric_suffixes = {
+        SLVCols.employees: "emp",
+        SLVCols.starters: "strt",
+        SLVCols.leavers: "stop",
+        SLVCols.vacancies: "vacy",
+    }
+
+    label_structs = [
```
Comment (RoyPriceCS50) [inline projects/_07_workforce_characteristics/_01_starters_leavers_vacancies/fargate/utils/prepare_utils.py:163]: This is rather cryptic. Add a note?

File: projects/_07_workforce_characteristics/_01_starters_leavers_vacancies/fargate/validate_00_prepare.py [hunk 2]
```diff
@@ -3,7 +3,6 @@
 import pointblank as pb
 import polars as pl
```
Comment (RoyPriceCS50) [inline projects/_07_workforce_characteristics/_01_starters_leavers_vacancies/fargate/validate_00_prepare.py]: Unused import

**Review by RoyPriceCS50 — APPROVED** [review]
Looks good. Just pop a note on the labels_structs showing what is creates, and an unused import can do.

Merge conflict to solve too. Guessing it's Achyut's join change.
